### ISSUE TITLE
Remove cl_interp, as it was supposed to be

### ISSUE THREAD

**Issue by AnAkkk** [ValveSoftware/Source-1-Games#816]
Since the OB Engine/TF2 release, cl_interp was put back (it was previously removed and replaced by cl_interp_ratio in EP1 Engine), and now there are both cl_interp and cl_interp_ratio to set the interpolation time. I don't think this was intended as the plan was to set the interpolation time through cl_interp_ratio (with the relation cl_interp_ratio/cl_updaterate = interp time).

See here: http://store.steampowered.com/news/907/
And here is an email from Alfred Reynolds to the HLDS mailing list about it: http://www.mail-archive.com/hlds@list.valvesoftware.com/msg21416.html

sv_client_max_interp_ratio and sv_client_min_interp_ratio were supposed to set the limits for cl_interp_ratio.
Since cl_interp was added back, this is useless as cl_interp_ratio only limits the minimum value that cl_interp can have. That means sv_client_max_interp_ratio can be set to any value, but it'll be useless (it won't set the cl_interp max value).

Since we can't limit the interp anymore, this lead to some exploiting:
http://forums.steampowered.com/forums/showthread.php?t=3129310
http://youtu.be/qBXQwmazf-o

TLDR
<b>cl_interp should be removed and the interpolation time should be set through cl_interp_ratio</b>
(or if you can, make the game automatically adjust the interp correctly, without the need for players to change it)

EDIT:
BTW, I've already emailed this to some Valve developers back in 2010, one answered:

<pre>I haven't looked at the cl_interp/interp_ratio thing, sounds like an oversight.</pre>
